src/main/java/com/exadel/discount/entity/VendorLocation.java

```diff
@@ -0,0 +1,42 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "vendor_locations")
+public class VendorLocation {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    @Column(name = "id")
+    private UUID id;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "vendor_id", nullable = false)
+    private Vendor vendor;
+
+    @Column(name = "country_id")
+    private UUID countryId;
+    @Column(name = "city_id")
+    private UUID cityId;
+    @Column(name = "contact", length = 50)
+    private String contact;
+    @Column(name = "coordinates")
+    private String coordinates;
+
```

Comment:
Exactly the same column name postgresql makes itself automatically.
Think , annotations @Column( are redundant in some  fields.

Reply:
I think either all annotations or none

Reply:
Agree) Done

Reply:
U're right. So it's decided to annotate every field.